lib/normalizer.c

```diff
@@ -845,6 +845,13 @@ grn_nfkc_normalize_expand(grn_ctx *ctx,
                                     "");
 }
 
+grn_inline static const unsigned char *
+grn_nfkc_normalize_unify_alphabet_diacritical_mark(
+  const unsigned char *utf8_char, unsigned char *unified)
+{
+  return utf8_char;
```

Comment:
```suggestion
  // TODO: We should return `unified` from this function. However, this function does not implement yet.
  //              I will use `unified` instead of `utf8_char` when I implement this function.
  return utf8_char;
```

Reply:
fix: [946d5fd](https://github.com/groonga/groonga/pull/1831/commits/946d5fdaa4230dd8aedd6957584c77071809a5eb) Thanks!